unity-renderer/Assets/DCLPlugins/ECS7/ECS7Plugin.cs

```diff
@@ -51,6 +62,25 @@ public void Dispose()
             systemsController.Dispose();
             internalEcsComponents.Dispose();
             crdtExecutorsManager.Dispose();
+
+            sceneController.OnNewSceneAdded -= SceneControllerOnOnNewSceneAdded;
+            sceneController.OnSceneRemoved -= SceneControllerOnOnSceneRemoved;
+        }
+
+        private void SceneControllerOnOnNewSceneAdded(IParcelScene scene)
```

Comment:
I wonder why this logic is here and not in `SceneControllerService`?

Reply:
Yeah that makes sense, I ended up moving the ecs7 loaded scenes tracking into the `SceneController` itself, tackled [here](https://github.com/decentraland/unity-renderer/pull/3700/commits/e4977d52e70286437de64a0654e41f93e217ca1e)